packages/components/data-table-manager/src/data-table-manager.tsx

```diff
@@ -6,14 +6,97 @@ import {
   type MouseEventHandler,
 } from 'react';
 import Spacings from '@commercetools-uikit/spacings';
-import DataTableSettings, {
-  type TDataTableSettingsProps,
-} from './data-table-settings';
+import DataTableSettings from './data-table-settings';
 
 export interface TRow {
   id: string;
 }
 
+export type TColumnData = {
+  key: string;
+  label: ReactNode;
+};
+
+export type TDisplaySettingsProps = {
```

Comment:
It seems now these types are duplicated here and it their _owning_ component (`DataTableSettings`). I guess this is a requirement for the readme generator to find the definitions but it feels somehow unwanted to duplicate our types because of a tool.

Perhaps as a follow up we could have a new task to investigate how we could avoid duplicating the types.

Reply:
💯

Reply:
The readme generator can only parse the AST of this file, as `react-docgen` is not able to "look" into other files.

<img width="890" alt="image" src="https://user-images.githubusercontent.com/1110551/182350957-a28031df-f6ae-4fc5-9530-584e2206c186.png">

Not sure how much we can do about it...

Reply:
We could maybe include multiple React components (in this case also `DataTableSettings`) so that we document both.

See example:
https://github.com/commercetools/ui-kit/blob/9a09eeec1ff7ab4b0bd95a4f0317ee589743fada/packages/components/primary-action-dropdown/package.json#L44

Reply:
Thanks, I added that here 0d0c0f2e7a5728fe6d8b8ad071e3f66ee02a4974

Reply:
It seems like we would need to adjust the signature reference linking, in case we include multiple components and the component matches one of the signatures.

For example `displaySettings?: TDisplaySettingsProps` is rendered as
<img width="1081" alt="image" src="https://user-images.githubusercontent.com/1110551/183450100-c441d856-52dc-417f-9bd9-79f05348f0c1.png">

But I think we want it to be linked to `DataTableSettings`, right?

Reply:
Many thanks for noticing that 👍 I updated that in 95e18767375c114da8e17c608fb65386747f3f7b
This works for GitHub readmes, however, I guess links to headings don't work in storybook
https://ui-lublrlj8z-commercetools.vercel.app/?path=/story/components-datatable--datatablemanager
(for all of the components I checked, not only this one)

Reply:
Thanks 👍 I will change that. The script still needs some work related to `--all-workspace-packages` option

Reply:
Thanks again @CarlosCortizasCT for finding the solution 💯 I updated the script and all readme-s in 7b4b9e7eaccbd1acd86f88651eb0e0636d04e185